governance.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Governance 1.0 DRAFT WIP
+description: Governance 1.0 discussion draft
+---
+# Governance 1.0 `DRAFT WIP`
+
+This document contains a work in progress (WIP) draft of the initial governance of the Op1st project (Governance 1.0).
+
+It also has a roadmap for additions coming in the next iteration of governance (Governane 1.1).
+
+## Op1st Community Governance 1.0 (MVG)
+
+A healthy open source project is one that demonstrates open practices, uses open infrastructure, and cultivates an open culture with the goal of becoming more sustainable.
+
+### Principles
+As a new cloud-native community growing up at the footsteps of massively scaled and successful projects such as Kubernetes and OpenInfra, we want to be careful to learn from those who came before us, while forging our own path into the challenges we've chosen to undertake.
+
+To stand up this Governance for a 1.0 good-enough minimal-viable-governance (MVG), we are looking to our key predecessors for some of our starting points.
+As an initial set of principles we are combining elements of the [Kubernetes community principles](https://github.com/kubernetes/community/blob/master/governance.md#principles) and the [OpenInfra Foundations Four Opens](https://openinfra.dev/four-opens/).
+
+#### Op1st Community Principles 1.0
+Our mission requires us to use open source principles in ways beyond licensing the code we write, but at the core we are an open source community and want to adhere to the best possible principles for creating an open, welcoming, and inclusive community.
+
+These are the Op1st community principles we adhere to:
+1. **Open Source** : Any content we create is released under an open source license. Content includes software, documentations, configuration files, and training materials.
+  - This is crucial to fulfilling our mission around open sourcing operations
+2. **Welcoming and respectful** : We intend to be an open, welcoming, and inclusive community.
```

Comment:
Thanks for thinking of this and suggesting it. I'm going to provide all my context here so everyone can understand, add to, and share the reasoning.

I pulled the three-item turn of phrase together out of a sort-of modern sentiment—it's similar to wording used in projects perceived as inclusive. The general approach is to focus on [being as inclusive as possible when designing an architecture for making a more diverse and equitable community](https://www.theopensourceway.org/the_open_source_way-guidebook-2.0.html#_to_build_diverse_open_source_communities_make_them_inclusive_first). When people come to the community and find it accessible, regardless of who they are, they can more easily and willingly bring their voices to the common table.

But recently I've been thinking about how to bring _equitability_ into these processes, statements, and outcomes, especially since the [All In Open Source](https://allinopensource.org) project started with the goal of open sourcing diversity, equity, and inclusion. I figure one of the first steps is to talk about equity, and I appreciate you seeing it here as an ideal we put forth. That means it's something we talk about in the SIGs—how to be more equitable, how to measure our success at that, how to improve—and thus something we can improve on over time as the project grows more diverse.

```suggestion
2. **Welcoming and respectful** : We intend to be an open, welcoming, equitable, and inclusive community.
```

Reply:
@akrawczy do you have any preferences of these variations?